pom.xml

```diff
@@ -47,10 +47,10 @@
     </distributionManagement>
 
     <properties>
-        <java.version>1.7</java.version>
-        <logback.version>1.2.0</logback.version>
-        <junit.version>4.13.1</junit.version>
-        <jackson-databind.version>2.10.0.pr1</jackson-databind.version>
+        <java.version>1.8</java.version>
```

Comment:
doesn't seem that any of the updated dependencies would require java 8 upgrade (jackson-databind says java 7 works for it until `2.12`, and the rest are just revision updates).. 

do we really need to drop support for java 7 just for these deps updates? i'm fine with updating the deps further (their major versions), but seems like we wouldn't capitalize over such change in this PR..

Reply:
No, we can absolutely keep Java 7, seems like the best thing to do.
Fixed!